src/main/java/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -41,6 +41,15 @@ protected ResponseEntity<Object> handleEntityNotFoundException(
                 .body(ex.getMessage());
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex
+    ) {
+        return ResponseEntity
+                .status(HttpStatus.BAD_REQUEST)
```

Comment:
you throw this exception when there is duplicate email

in this case you should use HTTP 409 CONFLICT